### ISSUE TITLE
brew install gor - requires libpcap-dev

### ISSUE THREAD

**Issue by black-snow** [Homebrew/linuxbrew-core#14837]
- [X] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)
- [X] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [X] ran `brew update` and can still reproduce the problem?
- [X] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [-] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [-] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:

- What you were trying to do (and why)

`brew install gor`

- What happened (include command output)

```
Last 15 lines from /home/xxx/.cache/Homebrew/Logs/gor/01.go:
go: extracting github.com/pierrec/lz4 v0.0.0-20161206202305-5c9560bfa9ac
go: downloading github.com/pierrec/xxHash v0.0.0-20160112165351-5a004441f897
go: extracting github.com/pierrec/xxHash v0.0.0-20160112165351-5a004441f897
go: finding github.com/mattbaird/elastigo latest
go: downloading github.com/mattbaird/elastigo v0.0.0-20170123220020-2fe47fd29e4b
go: extracting github.com/mattbaird/elastigo v0.0.0-20170123220020-2fe47fd29e4b
go: finding github.com/araddon/gou latest
go: finding github.com/bitly/go-hostpool latest
go: downloading github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932
go: downloading github.com/araddon/gou v0.0.0-20190110011759-c797efecbb61
go: extracting github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932
go: extracting github.com/araddon/gou v0.0.0-20190110011759-c797efecbb61
# github.com/google/gopacket/pcap
../../../../pkg/mod/github.com/google/gopacket@v0.0.0-20190211013929-f86faeb88894/pcap/pcap_unix.go:34:18: fatal error: pcap.h: No such file or directory
compilation terminated.
```

- What you expected to happen

`gor` successfully installed.

- Step-by-step reproduction instructions (by running `brew install` commands)

`brew install gor`

----

I had to manually `apt install libpcap-dev` for the pcap header was missing. I don't know how to add this to the dependencies, otherwise I'd file a PR.

**Comment by issyl0** [comment]
Thanks for reporting this. Adding `depends_on "libpcap" unless OS.mac?` to the formula should do it. We look forward to your PR!

**Comment by issyl0** [comment]
This has been fixed by #14851. Thanks again, @black-snow!
